File: music-lessons-2-java/app/src/main/java/com/mstoyanov/musiclessons/model/Weekday.java
package com.mstoyanov.musiclessons.model;

import java.io.Serializable;

public enum Weekday implements Serializable {
    MONDAY("Monday"),
    TUESDAY("Tuesday"),
    WEDNESDAY("Wednesday"),
    THURSDAY("Thursday"),
    FRIDAY("Friday"),
    SATURDAY("Saturday"),
    SUNDAY("Sunday");

    String value;

    Weekday(String value) {
        this.value = value;
    }

    public String displayValue() {
        return value;
    }
}

File: app/src/main/kotlin/com/mstoyanov/music_lessons/model/Weekday.kt
package com.mstoyanov.music_lessons.model

import java.io.Serializable

enum class Weekday(private var value: String) : Serializable {
    MONDAY("Monday"),
    TUESDAY("Tuesday"),
    WEDNESDAY("Wednesday"),
    THURSDAY("Thursday"),
    FRIDAY("Friday"),
    SATURDAY("Saturday"),
    SUNDAY("Sunday");

    fun displayValue(): String {
        return value
    }
}
